layouts/blog/list.html

```diff
@@ -24,6 +24,9 @@ <h5 class="mt-0 mb-1"><a href="{{ .RelPermalink }}">{{ .LinkTitle }}</a></h5>
 					<p class="mb-2 mb-md-3">
 						<small class="text-muted">
 							{{ with .Params.author }}{{ T "post_byline_by" }} <b>{{ . | markdownify }}</b> |{{ end}}
+              {{ if .Params.translator }}
+                Translated {{ with .Params.translator }}{{ T "post_byline_by" }} <b>{{ . | markdownify }}</b> |{{ end }}
+              {{ end }}
```

Comment:
Again, we would need similar changes here to remove hardcoded strings and use only one check.

Reply:
fixed by https://github.com/kubernetes/website/pull/47270/commits/3444b2616332c0da9d2b0c9366c99d5b6698f1ef too!